composer.json

```diff
@@ -13,9 +13,8 @@
         "php": ">=5.4.0",
         "ext-openssl": "*",
         "ext-mcrypt": "*",
-        "ext-mbstring": "*",
```

Comment:
I don't get it... Support package definitely needs the ext, why would I remove it? Then, isn't the ext redundant in the main package?